ROMFS/CMakeLists.txt

```diff
@@ -121,7 +121,6 @@ add_custom_command(
 		${romfs_gen_root_dir}/init.d/rc.autostart
 		${romfs_gen_root_dir}/init.d/rc.autostart.post
 		${romfs_copy_stamp}
-	COMMAND ${CMAKE_COMMAND} -E remove_directory ${romfs_gen_root_dir}/*
```

Comment:
I'm not sure, did you verify this is still working cleanly with ROMFS changes and incremental rebuilds?

Reply:
I believe the proper solution to all of this would be to have 1-1 dependencies for every file, then everything would actually work as intended, including `clean`, and other subtle dependencies.

The blocker is that airframes can be tagged excluded (processed through python scripts) and the actual build system is oblivious to this.

Reply:
I assume it won't work cleanly with ROMFS changes. That's what I wrote in my message.

Reply:
I honestly don't know exactly the way you want to fix this all properly. It's all a bit of a mystery to me.

So, if you come with an alternative fix in the next few days, fine, otherwise I suggest we merge this, or remove the param and the functionality.

Reply:
Would we be able to make the ability to add files in a board init folder automatic? If it exits copy it in to the romfs. Then a script in the boards init files could reference the files and allow for a more structured init, and shared init files.

Reply:
Alright, I'll leave this to you to fix then.

Reply:
@dagar reminder that you wanted to fix this properly...